Cargo.toml

```
@@ -18,3 +18,6 @@ winapi = { version = "0.3", features = ["winbase"] }
 
 [dev-dependencies]
 diff = "0.1.10"
+
+[dependencies]
```

This dependency should be Unix only, see [this section](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#platform-specific-dependencies) of the Cargo book for a how to
```suggestion
[target.'cfg(unix)'.dependencies]
```